### ISSUE TITLE
ceval和mmlu关于ABCD不同的解码

### ISSUE THREAD

**Issue by msn199959** [QwenLM/Qwen#312]
### 是否已有关于该错误的issue或讨论？ | Is there an existing issue / discussion for this?

- [X] 我已经搜索过已有的issues和讨论 | I have searched the existing issues / discussions

### 该问题是否在FAQ中有解答？ | Is there an existing answer for this in FAQ?

- [X] 我已经搜索过FAQ | I have searched FAQ

### 当前行为 | Current Behavior

Qwen-7B-main/eval/evaluate_mmlu.py里面是
softval = torch.nn.functional.softmax(
            torch.tensor(
                [
                    logits[tokenizer(" A")["input_ids"]],
                    logits[tokenizer(" B")["input_ids"]],
                    logits[tokenizer(" C")["input_ids"]],
                    logits[tokenizer(" D")["input_ids"]],
                ]
            ),
            dim=0,
        )
[' A']
但是Qwen-7B-main/eval/evaluate_ceval.py里面是
softval = torch.nn.functional.softmax(
            torch.tensor(
                [
                    logits[tokenizer("A")["input_ids"][1]],
                    logits[tokenizer("B")["input_ids"][1]],
                    logits[tokenizer("C")["input_ids"][1]],
                    logits[tokenizer("D")["input_ids"][1]],
                ]
            ),
            dim=0,
        )
[' A']比"A"多了个空格，我更换了个模型的tokenizer就会有不同的输出，请问是否有影响？

### 运行环境 | Environment

```Markdown
- OS:
- Python:
- Transformers:
- PyTorch:
- CUDA (`python -c 'import torch; print(torch.version.cuda)'`):
```

**Comment by Losin** [comment]
千问词表里' A'和'A'是不同的token，你可以试试把few-shot里的答案部分tokenize看看token ID，这里主要是为了和few-shot的token ID对齐。

你“更换了个模型的tokenizer”是指？
